Resources/ServerInfo/Guidebook/Antagonist/Revolutionaries.xml

```diff
@@ -40,7 +40,7 @@
   - [bold]Visibly be destroyed upon being implanted into a [color=#5e9cff]Head Revolutionary[/color][/bold], giving you away
   - NOT protect against flash disorientation
 
-  Assume all of [color=#cb0000]Security[/color] and [color=#1b67a5]Command[/color] are implanted with mindshields already.
+  Assume all of [color=#cb0000]Security[/color] and [color=#1b67a5]Command[/color] are implanted with mindshields already, [bold]however they can be removed using an empty implanter.[/bold]
```

Comment:
Might be nice to clarify `however they can be removed using an empty implanter, obtainable from the Medical department's MedFab.`